ocs_ci/ocs/node.py

```diff
@@ -1034,6 +1034,7 @@ def delete_and_create_osd_node_vsphere_upi_lso(osd_node_name, use_existing_node=
     )
 
     osd_id = osd_ids[0]
+    assert osd_id, "osd ids not found"
```

Comment:
It would be better to not use assert in modules. It is better to raise an exception that could be handled in test based on context.

Reply:
> It would be better to not use assert in modules. It is better to raise an exception that could be handled in test based on context.

ack, thx
the assert will be removed, it is redundant